pom.xml

```diff
@@ -1,14 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
-	<modelVersion>4.0.0</modelVersion>
-	<groupId>io.mosip.wuri</groupId>
-	<artifactId>registration</artifactId>
-	<version>1.1.1</version>
-	<packaging>pom</packaging>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.mosip.wuri</groupId>
+    <artifactId>registration</artifactId>
+    <version>1.1.2</version>
```

Comment:
Pourquoi 1.1.2 ?

Reply:
We already discuss about 1.1.2 version, at the end we will back to 1.1.1